src/spikeinterface/core/generate.py

```diff
@@ -173,7 +173,7 @@ def generate_sorting(
             duration=durations[segment_index],
             refractory_period_ms=refractory_period_ms,
             firing_rates=firing_rates,
-            seed=seed,
+            seed=seed + segment_index,
```

Comment:
oups. good catch. maybe we could have something to generate several seeds we several segment. but ouside the scope of this PR